tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc

```diff
@@ -8754,7 +8754,7 @@ TEST_F(EnergyPlusFixture, UnitarySystem_ASHRAEModel_WaterCoils)
     SimUnitarySystem(UnitarySystem(1).Name, FirstHVACIteration, UnitarySystem(1).ControlZoneNum, ZoneEquipList(1).EquipIndex(1), _, _, _, _, true);
 
     ZoneTemp = Node(ControlZoneNum).Temp;
-    CpAir = PsyCpAirFnWTdb(Node(InletNode).HumRat, Node(InletNode).Temp);
+    CpAir = PsyCpAirFnW(Node(InletNode).HumRat, Node(InletNode).Temp);
```

Comment:
Same as above. There are a few places that need removing the temp argument from the PsyCpAirFnW() function.